docs/1-1-planning.md

```diff
@@ -25,13 +25,13 @@ There were other reasons for using Parse - much of the content displayed within
 
-As React Native doesn't need to be tightly connected to a data layer, for example development of the UI and app logic in a React Native app can be done with simple mock data. This means that as long as the structure of the data remains the same, you can swap the data source of a fully built app with minimal amounts of adjustment. For the F8 App this meant we could very easily transition from Parse Cloud Code to the open source Parse Server after the app had already been developed. We'll cover this more in the [data tutorial](/_data/tutorials.yml/).
+React Native doesn't need to be tightly connected to a data layer. For example development of the UI and app logic in a React Native app can be done with simple mock data. This means that as long as the structure of the data remains the same, you can swap the data source of a fully built app with minimal amounts of adjustment. For the F8 App this meant we could very easily transition from Parse Cloud Code to the open source Parse Server after the app had already been developed. We'll cover this more in the [data tutorial](/_data/tutorials.yml/).
 
 ### Data Access with React Native
 
-To get Parse and React Native working together, there is an existing [Parse + React package](https://github.com/ParsePlatform/ParseReact) that provides the necessary binding tools, but there was a problem - due to the vagaries of conference wi-fi and connectivity, the F8 app must be able to work offline. Since Parse + React did not support offline syncing of data when the F8 app was being built, so we had to construct our own offline support.
+To get Parse and React Native working together, there is an existing [Parse + React package](https://github.com/ParsePlatform/ParseReact) that provides the necessary binding tools, but there was a problem - due to the vagaries of conference wi-fi and connectivity, the F8 app must be able to work offline. Since Parse + React did not support offline syncing of data when the F8 app was being built, we had to construct our own offline support.
 
-There was another factor in making this decision - team size. Relay, for example, would be more appropriate for a larger team working at scale, but the F8 app was being developed by one person, with a few others in support for design. This has a big influence on the type of data access method you use in an app.
+There was another factor in choosing our data access layer - team size. Relay - a javascript framework for accessing data - for example, would be more appropriate for a larger team working at scale, but the F8 app was being developed by one person, with a few others in support for design. This had a big influence on the type of data access method we chose in the F8 app.
```

Comment:
How about...

"For example, Relay, a JavaScript framework for accessing data, would be more appropriate..."